examples/charts/file/dots.py

```diff
@@ -9,16 +9,19 @@
     jython=[22, 43, 10, 25, 26],
 )
 
-# any of the following commented are also valid Dot inputs
-#xyvalues = pd.DataFrame(xyvalues)
-#xyvalues = list(xyvalues.values())
-#xyvalues = np.array(list(xyvalues.values()))
-
 output_file("dots.html")
 
-dots = Dot(
-    xyvalues, cat=['lists','loops','dicts', 'gen exp', 'exceptions'],
-    title="Dots Example", ylabel='Performance', legend=True
-)
+from bokeh.charts import Bar, output_file, show, hplot
+
+# best support is with data in a format that is table-like
```

Comment:
Should probably reference the documentation if anything where it describes the accepted input types since table-like is a bit ambiguous, since charts tries to convert any set of inputs into a dataframe internally.